combat-trainer.lic

```diff
@@ -2215,6 +2215,11 @@ class TrainerProcess
     when 'App Bundle'
       retreat
       bput("app my bundle quick", 'You scan', 'Appraise what', 'You cannot appraise', 'You can\'t appraise')
+    when /Summon (?<element>Aether|Air|Earth|Electricity|Fire|Water)/
+      retreat
+      bput("summon #{Regexp.last_match[:element]}", 'turn your attention inward', 'Roundtime')
+      waitrt?
```

Comment:
Can a domain be ended/closed? If so, should we do that when we're done training in a room?

Reply:
Not that I'm aware of, at least, nothing stands out to me in the help text.

```
> summon
Summon allows Warrior Mages to draw down forces from the Elemental Planes.  Normally this takes the form of magical creatures, but Warrior Mages can learn to summon subtle and unlocalized expressions of the Elemental Planes' otherworldly physics.
  SUMMON ADMITTANCE (Universal)
A meditation that draws Elemental Charge from your aligned plane.
  SUMMON <ELEMENT> DOMAIN (Variable)
A ritual that attunes an area to an Elemental Plane, easing spells of the appropriate element.
  SUMMON FAMILIAR (Aether)
A ritual that summons an aether elemental to serve as your familiar.
  SUMMON IMPEDANCE (Universal)
A meditation that discharges Elemental Charge safely.
  SUMMON WEAPON (Earth)
A ritual that summons a weapon made of stone.
```

Reply:
I did notice that after a while I see this message, `"A strange sense passes through you.  Afterward, the area feels somehow more... normal than it did a moment ago."` -- I suppose that might be the summoned domain effect ending on its own?